spec/zelastic/indexer_spec.rb

```diff
@@ -3,24 +3,30 @@
 require 'spec_helper'
 
 RSpec.describe Zelastic::Indexer do
-  let(:type) { Gem::Version.new(client.info.dig('version', 'number')) <= Gem::Version.new('7.0.0') }
   let(:config) do
     Zelastic::Config.new(
       client: client,
       data_source: data_source,
       mapping: mapping,
-      type: type
+      logger: Logger.new('log/test.log')
```

Comment:
Are we writing to a log during testing?

Reply:
The Indexer and IndexManager do write to the logger and by default it would output to stdout during tests:
https://app.circleci.com/pipelines/github/carwow/zelastic/115/workflows/6ec74cf3-d8be-4ff4-b165-4bf3b625dc44/jobs/606

So basically to avoid a wall of test during tests.

Reply:
Isn't there's a null logger or something we could use

Reply:
We could log to `/dev/null` if we didn't care about the file

Reply:
It can still be useful for debugging specs. It is in a test so easy to change later if needed. This gem needs a lot more love.